### ISSUE TITLE
ReferenceError: exports is not defined

### ISSUE THREAD

**Issue by marcelokanzaki** [vercel/swr#24]
I'm getting this error after trying to import.

```javascript
import useSWR from '@zeit/swr'
```

**Comment by pacocoursey** [comment]
Would you provide some more information on your project setup? Are you using Next.js, CRA, or similar? Any configuration files would be helpful too.

This should work out of the box with Next.js if you want to get started quickly:

```
npm init next-app && yarn add @zeit/swr
```

**Comment by adamsanderson** [comment]
Since `webpacker` adds a layer of indirection, you might want to verify everything is in the right places.  I would double check that `@zeit/swr` is in the `node_modules` directory and listed in your `package.json` alongside react and other packages.

**Comment by siebertm** [comment]
We're running into this issue, too.

By default, `webpacker` configures babel to transpile `node_modules`, too. In *our* config, we disabled this, opting in to modules which provide ES6 code.

As `swr` uses arrow functions in it's `dist` modules, this breaks IE11, so we'd opt in to having babel transpile `swr`, too.

Now, something in the chain breaks and leads to `ReferenceError: exports is not defined` in the line

```js
Object.defineProperty(exports, "__esModule", {
  value: true
});
```

(dist/useSWR.js)

**Comment by jakobrosenberg** [comment]
Also getting this error.

Only fix is setting ``"version": 1`` in ``now.json``

**Comment by shuding** [comment]
This issue should have been fixed in v0.2.3.
